packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -827,15 +827,41 @@ def check_cid(registry):
 
     @retrying.retry(wait_fixed=STD_INTERVAL, stop_max_delay=LATENCY * 1000)
     def test_application_metrics(agent_ip, agent_id, task_name, num_containers):
+        # Get expected 2 container ids from mesos state endpoint
+        # (one container + its parent container)
+        @retrying.retry(wait_fixed=STD_INTERVAL, stop_max_delay=METRICS_WAITTIME)
+        def get_container_ids_from_state(dcos_api_session, num_containers):
```

Comment:
I think this function could be a method of `dcos_api_session` I suspect it's used more than one time